http-client/src/test/groovy/io/micronaut/http/client/netty/DefaultClientHeaderMaskTest.groovy

```diff
@@ -1,42 +1,29 @@
 package io.micronaut.http.client.netty
 
```

Comment:
Do we need this test anymore?

We used to test that the actual client masked logging, but now this just tests the utils class

Which we also test in [http/src/test/groovy/io/micronaut/http/util/HttpHeadersUtilSpec.groovy](https://github.com/micronaut-projects/micronaut-core/pull/8471/files#diff-75a2657025e80f8e9f2d653092d04a309f987455dc0bf67bc91b005d1babfb1e)

Reply:
do you prefer if I delete it?

Reply:
I'm wondering if we should put it back how it was to check the real client is calling the masking utils?

Reply:
how about https://github.com/micronaut-projects/micronaut-core/pull/8471/commits/dc42add597516c21f7548d6f60f35f84cc93375c ?

Reply:
How about https://github.com/micronaut-projects/micronaut-core/pull/8471/commits/86b52a10ffe1b52e0c471f06918f4739b309cef6 ?

To avoid the Stub and the private method?